api/provider/facebook.go

```diff
@@ -69,7 +81,10 @@ func (p facebookProvider) GetUserData(ctx context.Context, tok *oauth2.Token) (*
 			nameKey:      strings.TrimSpace(u.FirstName + " " + u.LastName),
 			avatarURLKey: u.Avatar.Data.URL,
 		},
-		Email:    u.Email,
-		Verified: true,
+		Emails: []Email{{
+			Email:    u.Email,
+			Verified: true,
+			Primary:  true,
```

Comment:
Does FB guarantee these are all primary and all verified?

Reply:
We only extract a single email from Facebook so it is marked as `Primary` for consistency (same with Google).
The verified flag was already set to `true` but let me verify if we can extract that information.